uniqueids/tests.py

```diff
@@ -760,7 +760,8 @@ def test_personnel_upload_good(self):
             {
                 'communicate_through': None,
                 'details': {
-                    'addresses': {'msisdn': {'+2347012312341': {"default": True}}},
+                    'addresses': {
```

Comment:
I'm assuming these were the missing environment variables that caused the previous fail?

Reply:
It was `flake8` complaining that the line was too long.

Reply:
okay great 👍